packages/build/tests/core/tests.js

```diff
@@ -563,16 +548,9 @@ test.serial('functions can have a config with name passed to zip-it-and-ship-it'
   const { functions: functions } = await importJsonFile(call1Args[2].manifest)
 
   t.is(functions[0].displayName, 'Function One')
+  t.is(functions[0].generator, '@netlify/mock-plugin@1.0.0')
   t.is(functions[1].displayName, undefined)
-})
 
-test.serial('zip-it-and-ship-it runs without error when loading json config files', async (t) => {
```

Comment:
Why are you removing this test? I thought this was testing the deploy configuration API?

Reply:
I made the same test a part of another test I wrote :) so now we have one test with both a snapshot and different field checks :)